spring-cloud-dataflow-docs/src/main/asciidoc/configuration-local.adoc

```diff
@@ -196,6 +196,173 @@ spring:
     driver-class-name:org.postgresql.Driver
 ----
 
+[start=4]
+. Or build a custom server with your own build files. Here are examples of bundling MySQL driver using gradle and maven, which
```

Comment:
It could be:

> Alternatively, you can build a custom server with your build files. Here are the examples of bundling MySQL driver using Gradle and Maven, which will result in the same jar as `2.1.2.RELEASE` with the addition of a `mysql-connector-java-8.0.16.jar`: